lib/src/components/NxDropdownMenu/NxDropdownMenu.scss

```diff
@@ -62,12 +62,12 @@
   width: 100%;
 
   &:hover {
-    background-color: var(--nx-color-interactive-background-hover);
+    background-color: var(--nx-color-dropdown-menu-background-hover);
     color: var(--nx-color-text-dark);
```

Comment:
In light mode, the text colour gets darker, but in dark mode the text colour of the the button stays the same. Maybe check with Jason to confirm that's correct? Also it's now `--nx-color-text-stark` 😄

Reply:
> In light mode, the text colour gets darker, but in dark mode the text colour of the the button stays the same (this is the same for the active state). Maybe check with Jason to confirm that's correct? Also it's now `--nx-color-text-stark` 😄

Oo I might've missed it but, for me, it looks like the text color is constant on the toggle button 🤔 
THanks! Changed to stark here: 9c6b042a322d38a14db3641edde860082a8ed87e

Reply:
> > In light mode, the text colour gets darker, but in dark mode the text colour of the the button stays the same (this is the same for the active state). Maybe check with Jason to confirm that's correct? Also it's now `--nx-color-text-stark` 😄
> 
> Oo I might've missed it but, for me, it looks like the text color is constant on the toggle button 🤔 THanks! Changed to stark here: [9c6b042](https://github.com/sonatype/sonatype-react-shared-components/commit/9c6b042a322d38a14db3641edde860082a8ed87e)

It's referring to the `.nx-dropdown-button` inside of the dropdown menu (not the toggle). So in light mode, the button's text is nx-color-text regularly, but when hovered, the text changes to nx-color-text-stark on hover. But in the dark mode mocks, the buttons in the dropdown do not change in colour when hovered or clicked.

Reply:
> > > In light mode, the text colour gets darker, but in dark mode the text colour of the the button stays the same (this is the same for the active state). Maybe check with Jason to confirm that's correct? Also it's now `--nx-color-text-stark` 😄
> > 
> > 
> > Oo I might've missed it but, for me, it looks like the text color is constant on the toggle button 🤔 THanks! Changed to stark here: [9c6b042](https://github.com/sonatype/sonatype-react-shared-components/commit/9c6b042a322d38a14db3641edde860082a8ed87e)
> 
> It's referring to the `.nx-dropdown-button` inside of the dropdown menu (not the toggle). So in light mode, the button's text is nx-color-text regularly, but when hovered, the text changes to nx-color-text-stark on hover. But in the dark mode mocks, the buttons in the dropdown do not change in colour when hovered or clicked.

Ah thanks, fixed here and confirmed with Jason
84a124a8986b6b13c3b07c33ef88260a6c803168